src/MCPClient/lib/clientScripts/upload-archivesspace.py

```diff
@@ -217,9 +219,10 @@ def upload_to_archivesspace(files, client, xlink_show, xlink_actuate, object_typ
     parser.add_argument('--uri_prefix', metavar="uri prefix")
     parser.add_argument('--access_conditions', metavar="conditions governing access", default="")
     parser.add_argument('--use_conditions', metavar="conditions governing use", default="")
+    parser.add_argument('--inherit_notes', metavar="Inherit digital object notes from the parent component", default=False, type=lambda x: x.lower() in ('yes', 'y', 'true', '1'))
```

Comment:
A better option here would be `action='store_true'` https://docs.python.org/2/library/argparse.html#action.  However, the dashboard config makes that difficult.  The `type` isn't for enforcing correctness, though.  `choices` is probably what you're looking for.  Also, the metavar is the stand in value for paramaters that take arguments (eg in `--foo BAR` BAR is the metavar).  The description should be in `help`